5.5/5.5.11/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 5.5.11
+создай объект с методом, которому мы передадим предложение (текст), а он вернет тоже предложение (текст),
+в котором слова будут отсортированы по возрастанию. Например "ямочка в доме" превратится в "в доме ямочка".
+
+Теперь добавь метод в этом объекте, который будет принимать слово и возвращать это слово с двумя заглавными
+буквами: вначале и вконце.
+
+Дальше добавить метод, который будет принимать текст и возвращать его, каждое слово в котором будет иметь эти 2
+заглавные буквы.
+
+Дальше создать метод, который будет удалять числа в словах в переданном тексте и потом возвращать его. Например эта
+фраза "до34м в о8г9не" должна превратиться в "дом в огне".
+
+В итоге в объект надо передать фразу "до34м в о8г9не", а он вернет "В ДоМ ОгнЕ".
+<script>
+	const infoObj = {
+		init(text) {
+			let newText = this.getTextWithoutNumbers(text);
+			newText = this.getSortAscending(newText);
+			newText = this.getInfo(newText);
+
+			return newText;
+		},
+		getNewArr(text) {
```

Comment:
Let's remove this method because we don't have enough benefits from using it:
```
const wordsArr = this.getNewArr(text);
```
or we can just use `split` directly
```
const wordsArr = text.split(' ');  // this code is smaller as well
```